chart/values.yaml

```diff
@@ -1,11 +1,9 @@
 replicaCount: 1
 
 image:
-  # see https://github.com/hjacobs/kube-downscaler/releases
-  repository: mtr.devops.telekom.de/caas/kube-downscaler
+  repository: ghcr.io/caas-team/py-kube-downscaler
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "23.2.0@sha256:4129e7e7551eb451ee2b43680ef818f3057304ad50888f79ec9722afab6c29ff"
+  tag: "{{ .Values.image.tag | default .Chart.AppVersion }}"  # Dynamically set from CI
```

Comment:
Is an image with the chart.AppVersion even created when the github actions run? We should build the image as well, when we create a new tag in the repository.

Reply:
Currently, the CI/CD process does not automatically create a Docker image tagged specifically with the chart.AppVersion. Mentioned it as well in #19